src/commands/logs.ts

```diff
@@ -34,111 +33,7 @@ inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'))
  */
 export default async (cmd: { json: boolean; open: boolean; setup: boolean; watch: boolean }) => {
   await checkIfOnline();
-  /**
```

Comment:
Did you just reorder this code, or did you also make edits to it? Hard to tell with the GitHub diff view.

Reply:
Mostly just extracted these inner functions to outside the main one, small edits noted below.